### PR TITLE
refactor(console): increase custom ui assets upload timeout to 5 mins

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Make request timeout in the `useApi()` hook configurable and increase custom UI assets upload request timeout to 5 mins.
Since the maximum allowed zip file size is 20MB, it might cause a timeout error on some poor network conditions.

## Testing

Locally

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

File: packages/console/src/components/CustomUiAssetsUploader/index.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ import FileIcon from '../FileIcon';
 
 import * as styles from './index.module.scss';
 
+const requestTimeout = 300_000; // 5 minutes
```
Comment (gao-sun) [inline packages/console/src/components/CustomUiAssetsUploader/index.tsx:16]: does the api have the same timeout?

Comment (charIeszhao) [reply packages/console/src/components/CustomUiAssetsUploader/index.tsx:16]: Good point. I didn't set it on the server side, will update

Comment (charIeszhao) [reply packages/console/src/components/CustomUiAssetsUploader/index.tsx:16]: Just confirmed we have no timeout set on the API side, so a [new task](https://linear.app/silverhand/issue/LOG-9693/set-1-min-timeout-to-all-apis) has been added and we'll merge this PR first.

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Pull request merged**